doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-subprocesses-ref.adoc

```diff
@@ -0,0 +1,74 @@
+[id='bpmn-subprocesses-ref_{context}']
+
+= BPMN2 subprocesses in process designer
+
+A subprocess is an activity that contains nodes. You can embed part of the main process within a subprocess. You can also include variable definitions within the subprocess. These variables are accessible to all nodes inside the subprocess.
+
+A subprocess must have one incoming connection and one outgoing connection. A terminate end event inside a subprocess ends the subprocess instance but does not automatically end the parent process instance. A subprocess ends when there are no more active elements in it.
```

Comment:
@csherrar, I think the first suggestion since it indicates a standard requirement: A subprocess must have at least one incoming connection and one outgoing connection.

Reply:
Done, changed the statement as:

_A subprocess must have at least one incoming connection and one outgoing connection._